packages/formation-react/src/components/MegaMenu/SubMenu.jsx

```diff
@@ -21,6 +21,7 @@ const SubMenu = ({
   mobileMediaQuery,
   smallDesktopMediaQuery,
   data,
+  id,
```

Comment:
I don't see where this id is coming from- was there another PR that added this?

Reply:
This ID came from the `<MenuSection />` component. I passed it down and made it a named prop.

Previously it was being applied to the button. It was needed on the target element so JAWS would properly move to the controlled element when users entered the key chord `Alt + JAWS_Key + M`. 

Line 66 of MenuSection.jsx: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/ebf5e2f531d9bc1a3cc307a5f73ec6700673b20f/packages/formation-react/src/components/MegaMenu/MenuSection.jsx#L66